src/frequenz/sdk/microgrid/component_graph.py

```diff
@@ -34,6 +34,8 @@
 
 _logger = logging.getLogger(__name__)
 
+# pylint: disable=too-many-lines
```

Comment:
Not crazy about this, but I also guess this is out of scope for this PR, as to fix it we probably need to refactor the whole component graph.

Reply:
We could move the new function somewhere else to avoid exceeding the 1K LoC

Reply:
Yeah, that's what I meant, I wouldn't just move stuff arbitrarily to silence the warning, that's what the disable directive is for, so I would only do it if we do a thoughtful refactor (and maybe it just needs to be big and we have to keep the check disabled).